### ISSUE TITLE
[BUG] (STOP button does not stop/reset the print status)

### ISSUE THREAD

**Issue by Dentrax** [MarlinFirmware/Marlin#23440]
### Did you test the latest `bugfix-2.0.x` code?

No, but I will test it now!

### Bug Description

Pressing the Stop button stops the printing, but it's impossible to print the same model since the progress/status does not reset. (Which it was working as expected in v1.0.1)

I just upgraded my Marlin from v1.0.1 to latest.

P.S: I used the corresponding configs: https://github.com/MarlinFirmware/Configurations/tree/2.0.9.3/config/examples/Creality/Ender-3%20V2

### Expected behavior

It should reset the print status.

### Actual behavior

It does not reset the print status. It seems like I clicked the _pause_ button and wanted to resume the print.

The only workaround to print the same model from the beginning is that rebooting the printer by shutting it down.

### Steps to Reproduce

1. Start a print
2. After some progress complete, press the "Stop" button
3. Try to select the same model again to print and see that progress is not changed

### Version of Marlin Firmware

2.0.9.3

### Printer model

Creality Ender 3 V2

### Electronics

Stock

### Bed Leveling

No Bed Leveling

### Your Slicer

Cura

### Host Software

SD Card (headless)

### Additional information & file uploads

https://user-images.githubusercontent.com/16493751/147976519-2757ef75-6b18-40a3-80e5-0ea57181093c.mp4

**Comment by thinkyhead** [comment]
Does it work properly with 2.0.9.2 … or 2.0.9.1 ?

**Comment by Dentrax** [comment]
> Does it work properly with 2.0.9.2 … or 2.0.9.1 ?

I directly jumped from `v1.0.1` so I'm not sure. 🤷‍♂️

**Comment by thinkyhead** [comment]
Well if you get a chance to try either one out, please report your results. I am quite behind in dealing with issues and submissions at this time, so I will not able to devote any time to in-depth testing this week.

**Comment by descipher** [comment]
Is that screen still closed source? If so we can't do anything with it ..

**Comment by Dentrax** [comment]
> Is that screen still closed source? If so we can't do anything with it ..

By saying "closed source", you mean `T5UIC1_V20_4页面_191022.BIN` file that included in [DWIN_SET](https://github.com/MarlinFirmware/Configurations/tree/import-2.0.x/config/examples/Creality/Ender-3%20V2/DWIN_SET)?

**Comment by The-EG** [comment]
Which UI is this? It would be one of:
```c++
#define DWIN_CREALITY_LCD             // Creality UI
//#define DWIN_CREALITY_LCD_ENHANCED  // Enhanced UI
//#define DWIN_CREALITY_LCD_JYERSUI
```

with the default in the example being 'Creality UI'.

**Comment by The-EG** [comment]
> Is that screen still closed source? If so we can't do anything with it ..

Not exactly, this DWIN is much simpler than others. Assets (images, icons) are stored on the LCD itself and it can be commanded to display them, via serial interface. There are no built in menus, dialogs, etc.

There are currently 4 different UIs for this screen, however, and the UI pictured could be the stock 'CreaityUI' or either of the two derivatives, 'Enhanced' or 'Jyers'. Assuming this is a bug, the fix will need to be performed for the UI in question.

**Comment by descipher** [comment]
> > Is that screen still closed source? If so we can't do anything with it ..
> 
> Not exactly, this DWIN is much simpler than others. Assets (images, icons) are stored on the LCD itself and it can be commanded to display them, via serial interface. There are no built in menus, dialogs, etc.
> 
> There are currently 4 different UIs for this screen, however, and the UI pictured could be the stock 'CreaityUI' or either of the two derivatives, 'Enhanced' or 'Jyers'. Assuming this is a bug, the fix will need to be performed for the UI in question.

Very good, there are some encrypted ones .. you can't work with them.

**Comment by mriscoc** [comment]
> Which UI is this? It would be one of:
> 
> ```c++
> #define DWIN_CREALITY_LCD             // Creality UI
> //#define DWIN_CREALITY_LCD_ENHANCED  // Enhanced UI
> //#define DWIN_CREALITY_LCD_JYERSUI
> ```
> 
> with the default in the example being 'Creality UI'.

Seems to be the `DWIN_CREALITY_LCD` flavor.

Add this:
![image](https://user-images.githubusercontent.com/2745567/148344552-481f1a58-5436-493d-a891-2fd0596d3b2b.png)
To the dwin.cpp file

**Comment by thisiskeithb** [comment]
Fixed in #23481
